src/services/housing.js

```diff
@@ -208,7 +257,9 @@ const changeApartmentAppEditor = async (applicationID, newEditorUsername) => {
  */
 const getApartmentApplication = async (applicationID) => {
   try {
-    return await http.get(`housing/apartment/applications/${applicationID}/`);
+    let applicationResult = await http.get(`housing/apartment/applications/${applicationID}/`);
+    setApplicationDetails(applicationResult);
```

Comment:
Does this do anything if you do not catch its output?

Reply:
(Is applicationDetails defined outside the context  of setApplicationDetails?)

Reply:
No it is not, which is actually intentional in this case

Reply:
I have done a lot more testing of this feature in #1100  and have been able to verify its effectiveness